tests/phpunit/integration/api/html/beansBuildSkipLinks.php

```diff
@@ -0,0 +1,221 @@
+EOB;
+		ob_start();
+		beans_build_skip_links();
+		$actual = ob_get_clean();
+
+		$this->assertContains( $expected, $actual );
+	}
+
+	/**
+	 * Test beans_build_skip_links() should output content and primary sidebar skip links.
+	 */
+	public function test_should_output_content_primary_sidebar_skip_links() {
+		Monkey\Functions\expect( 'beans_is_active_widget_area' )
+		->once()
+		->with( 'sidebar_primary' )
+		->andReturn( 'true' );
+
+		add_filter( 'beans_default_layout', function( $default_layout ) {
+			return 'c_sp';
+		});
+
+		$this->assertEquals( beans_get_layout(), 'c_sp' );
+
+		$expected = <<<EOB
+<ul class="beans-skip-links"><li ><a href="#beans-content" class="screen-reader-shortcut">Skip to the content.</a></li><li ><a href="#beans-primary-sidebar" class="screen-reader-shortcut">Skip to the primary sidebar.</a></li></ul>
+EOB;
+		ob_start();
+		beans_build_skip_links();
+		$actual = ob_get_clean();
+
+		$this->assertContains( $expected, $actual );
+	}
+
+	/**
+	 * Test beans_build_skip_links() should output content and secondary sidebar skip links.
+	 */
+	public function test_should_output_content_secondary_sidebar_skip_links() {
+		Monkey\Functions\expect( 'beans_is_active_widget_area' )
+		->once()
+		->with( 'sidebar_secondary' )
+		->andReturn( 'true' );
+
+		add_filter( 'beans_default_layout', function( $default_layout ) {
+			return 'c_ss';
+		});
+
+		$this->assertEquals( beans_get_layout(), 'c_ss' );
+
+		$expected = <<<EOB
+<ul class="beans-skip-links"><li ><a href="#beans-content" class="screen-reader-shortcut">Skip to the content.</a></li><li ><a href="#beans-secondary-sidebar" class="screen-reader-shortcut">Skip to the secondary sidebar.</a></li></ul>
+EOB;
+		ob_start();
+		beans_build_skip_links();
+		$actual = ob_get_clean();
+
+		$this->assertContains( $expected, $actual );
+	}
+
+
+	/**
+	 * Test beans_build_skip_links() should output content and both sidebar skip links.
+	 */
+	public function test_should_output_content_both_sidebar_skip_links() {
+		Monkey\Functions\expect( 'beans_is_active_widget_area' )
+		->once()
+		->ordered()
+		->with( 'sidebar_primary' )
+		->andReturn( true )
+		->andAlsoExpectIt()
+		->once()
+		->with( 'sidebar_secondary' )
+		->andReturn( true );
+
+		add_filter( 'beans_default_layout', function( $default_layout ) {
+			return 'c_sp_ss';
+		});
+
+		$this->assertEquals( beans_get_layout(), 'c_sp_ss' );
+
+		$expected = <<<EOB
+<ul class="beans-skip-links"><li ><a href="#beans-content" class="screen-reader-shortcut">Skip to the content.</a></li><li ><a href="#beans-primary-sidebar" class="screen-reader-shortcut">Skip to the primary sidebar.</a></li><li ><a href="#beans-secondary-sidebar" class="screen-reader-shortcut">Skip to the secondary sidebar.</a></li></ul>
+EOB;
+		ob_start();
+		beans_build_skip_links();
+		$actual = ob_get_clean();
+
+		$this->assertContains( $expected, $actual );
```

Comment:
Here, you want to run both the expected and actual HTML strings through the `$this->format_the_html( )` method like this:

```
$this-> assertContains( $this->format_the_html( $expected ), $this->format_the_html( $actual ) );
```